scripts/run_e2e_suites_for_cov.sh

```diff
@@ -0,0 +1,38 @@
+
+export COVERAGE_MODE=true
+export LEDGER_SIM=true
+
+rm -rf coverage/e2e
+killall avalanchego
```

Comment:
this should be avoided, is a cleanup I added for easy repeated tests but may kill undesired processes